src/plays/dad-jokes/components/singleJoke/SingleJoke.css

```diff
@@ -0,0 +1,226 @@
+/*----------------------------Single Joke Container----------------------------*/
```

Comment:
Same comment as given to other CSS file.

Reply:
Done